angular/projects/admin-nrpti/src/app/mines/mines-add-edit/mines-add-edit.component.html

```diff
@@ -1,59 +1,149 @@
 <main class="container-fluid-padding">
-  <div class="spinner-container" *ngIf="loading">
-    <div class="spinner-new rotating"></div>
-  </div>
-  <div class="d-flex">
-    <h2>Edit Mine Details</h2>
-    <div class="ml-auto">
-      <button class="btn btn-outline-primary mr-1" type="cancel" (click)="navigateBack()">Cancel</button>
-      <button class="btn btn-primary" type="cancel" (click)="submit()">Save</button>
-    </div>
-  </div>
-  <hr>
-    <form *ngIf="!loading" [formGroup]="myForm" novalidate>
-    <div class="row">
-      <div class="label-pair mr-5 mt-4 col-12">
-        <label for="type" class="font-weight-bold mr-1">Type:</label>
-        <select [compareWith]="compareTypeSelection" name="type" id="type" formControlName="type" class="form-control">
-          <option *ngFor="let type of types" [ngValue]="type">{{ type }}</option>
-        </select>
+  <form *ngIf="!loading" [formGroup]="myForm" novalidate>
+    <section class="mb-3 pb-3 border-bottom">
+      <div class="d-flex">
+        <div>
+          <h2 class="border-0">Mine Information</h2>
+          <span class="grey-subtext ml-2">{{ lastEditedSubText }}</span>
+        </div>
+        <div class="ml-auto">
+          <div class="d-flex flex-column">
+            <mat-slide-toggle
+              name="publish"
+              id="publish"
+              class="publish-toggle ml-auto"
+              [ngClass]="myForm.get('publish').value ? 'published' : 'unpublished'"
+              formControlName="publish"
+              (change)="togglePublish($event)"
+              [checked]="myForm.get('publish').value"
+              title="{{ myForm.get('publish').value ? 'Unpublish mine' : 'Publish mine' }}"
+              ngDefaultControl
+            >
+              {{ (myForm.get('publish').value && 'Published') || 'Unpublished' }}
+            </mat-slide-toggle>
+            <span class="grey-subtext ml-auto">
+              *Changes made will take effect once you click "Update Record"
+            </span>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="my-3 py-3 border-bottom">
+      <div class="row">
+        <div class="col-sm col-md">
+          <div class="flex-container mb-2">
+            <label for="permittee" class="med mr-2">Permittee:</label>
+            <span id="permittee" name="permittee">{{ (mine && mine.permittee) || '-' }}</span>
+          </div>
+          <div class="flex-container mb-2">
+            <label for="status" class="med mr-2">Operation Status:</label>
+            <span id="status" name="status">{{ (mine && mine.status) || '-' }}</span>
+          </div>
+        </div>
+        <div class="col-sm col-md">
+          <div class="flex-container mb-2">
+            <label for="permit" class="med mr-2">Mine Act Permit Number:</label>
+            <span id="permit" name="permit">{{ (mine && mine.permitNumbers) || '-' }}</span>
```

Comment:
permitNumbers is an array. Might want to `.join(', ')` the output. Likely you can copy the whole header from the PR for the details screen that @MaxWardle was working on.

Reply:
Updated to use join as mentioned